docs/source/software_process.rst

```diff
@@ -0,0 +1,64 @@
+..  _software_process:
+
+================
+Software Process
+================
+
+----------------------
+Continuous Integration
+----------------------
+
+PyNWB is tested against Ubuntu, macOS and Windows operating systems.
+The project has both unit and integration tests.
+
+* Circleci runs all PyNWB tests against Ubuntu
+* Appveyor runs all PyNWB tests against Windows
+* Travis runs all PyNWB tests against macOS
+
+Tests are ran everytime there is a new branch. That way the contributor knows
```

Comment:
Tests are ran everytime there is a new branch -> Each time a PR is published or updated, the project is built, packaged and tested on all support operating systems and python distributions.